jcloud/api.py

```diff
@@ -215,7 +221,9 @@ async def remove(args):
                 print('[cyan]No worries. Exiting...[/cyan]')
                 return
 
-        _raw_list = await _list_by_phase(Phase.Serving.value, name='')
+        _raw_list = await _list_by_phase(
```

Comment:
Update the comment `# Case 3: remove all ALIVE flows.` to `# Case 3: remove all ALIVE and DELETED flows.`

Reply:
will do